website/docs/r/rds_cluster.html.markdown

```diff
@@ -191,7 +191,7 @@ The following arguments are supported:
 * `scaling_configuration` - (Optional) Nested attribute with scaling properties. Only valid when `engine_mode` is set to `serverless`. More details below.
 * `serverlessv2_scaling_configuration`- (Optional) Nested attribute with scaling properties for ServerlessV2. Only valid when `engine_mode` is set to `provisioned`. More details below.
 * `skip_final_snapshot` - (Optional) Determines whether a final DB snapshot is created before the DB cluster is deleted. If true is specified, no DB snapshot is created. If false is specified, a DB snapshot is created before the DB cluster is deleted, using the value from `final_snapshot_identifier`. Default is `false`.
-* `snapshot_identifier` - (Optional) Specifies whether or not to create this cluster from a snapshot. You can use either the name or ARN when specifying a DB cluster snapshot, or the ARN when specifying a DB snapshot.
+* `snapshot_identifier` - (Optional) Specifies whether or not to create this cluster from a snapshot. You can use either the name or ARN when specifying a DB cluster snapshot, or the ARN when specifying a DB snapshot. Conflicts with `global_cluster_identifier`. Clusters cannot be restored from snapshot **and** joined to an existing global cluster in a single operation. See the [AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-global-database-getting-started.html#aurora-global-database.use-snapshot) or this [`aws_rds_global_cluster` example](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_global_cluster#new-global-cluster-from-existing-db-cluster) for instructions on building a global cluster starting from a snapshot.
```

Comment:
The Terraform example linked doesn't seem to be an example of creating a cluster from a snapshot. If it's similar, the example should say that it's similar, or else a new example should be added

Reply:
Thanks for the feedback - I was on the fence about that Terraform example. It was helpful while researching alternatives for "restoring" global RDS clusters, but I think it's missing too much of the snapshot-related context.

[This comment](https://github.com/hashicorp/terraform-provider-aws/issues/29187#issuecomment-1477931892) on the original issue has a more targeted example, and perhaps should be migrated to this document for a proper example to link here instead.

Reply:
Yes, that looks good